src/components/BookList.js

```diff
@@ -25,6 +25,7 @@ function BookList() {
           />
         ))}
       </ul>
+      <div className="create"> </div>
```

Comment:
👍 Great attention to detail! Just a couple of tweaks to bring your design even closer to the intended look:

- [ ] The color you've used (`#f5f6f5`) for this line is slightly different from the design's color (`#e8e8e8`). Please consider updating your code to match the design's color for a consistent appearance.

- [ ] Additionally, please ensure that the weight layout matches the design more closely, as currently, they are not aligned with the design. These adjustments will help your project achieve a polished and professional appearance. Keep up the good work! 👍🎨

| Your | Design |
| --- | --- |
|  ![Frame 6](https://github.com/Nuga0909/bookstore3/assets/55840999/c007ac3c-ba99-43f7-bffc-a5815b9d468f) | ![image](https://github.com/Nuga0909/bookstore3/assets/55840999/8ae878a0-abb9-4a51-bd20-e4fcbcbd14cd) |